draft-ietf-quic-multipath.md

```diff
@@ -658,7 +658,9 @@ with Path ID 0.
 The PATH_NEW_CONNECTION_ID frame is used to issue new connection IDs for all paths.
 In order to let the peer open new paths, it is RECOMMENDED to proactively
 issue a Connection ID for at least one unused Path ID, as long as it remains
-compatible with the peer's Maximum Path ID limit.
+compatible with the peer's Maximum Path ID limit. An endpoint MAY issue at
+least one connection ID for all path IDs up to the peer's Maximum Path ID limit
+to avoid blocking.
 
```

Comment:
I would replace all that text by:

```
In order to let the peer open new paths, it is RECOMMENDED to proactively
issue at least one connection ID for all path IDs up to the minimum of the peer's Maximum Path ID
and the locally advertised Maximum Path ID.
```

Rationale: we are going to create confusion by recommending to allocate CID on only one path. The recommended way to not send too many CIDs is to lower the local path ID limit.